cores/esp8266/libb64/cencode.h

```diff
@@ -33,8 +33,6 @@ typedef struct {
 void base64_init_encodestate(base64_encodestate* state_in);
 void base64_init_encodestate_nonewlines(base64_encodestate* state_in);
 
-char base64_encode_value(char value_in);
```

Comment:
Removal of public function is a breaking change, see other comment.

Reply:
reverted